style.css

```diff
@@ -1,812 +1,707 @@
+  font-family: poppins, sans-serif;
+  border: 1px solid;
+}
 
-  a,
-  a::before,
-  a::after {
-    text-decoration: none;
-    color: inherit;
-  }
+h1,
+h2,
+h3 {
+  font-size: 32px;
+  font-weight: 700;
+  margin: 0;
+  color: black;
+}
 
-  p {
-    margin-top: 0;
-  }
+p {
+  margin-top: 0;
+}
 
-  .header {
-    color: #344563;
-    max-width: 100%;
-    padding: 16px;
-    margin: auto;
-    gap: 12px;
-    background-color: #fff;
-  }
+li {
+  list-style: none;
+}
 
-  .header-bar {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    max-width: 90rem;
-    margin: auto;
-    padding: 1rem;
-  }
+a {
+  text-decoration: none;
+  color: #344563;
+  margin-right: 12px;
+  font-size: 15px;
+  font-weight: 500;
+  line-height: 20px;
+}
 
-  .menu {
-    display: none;
-  }
+a,
+a::before,
+a::after {
+  text-decoration: none;
+  color: inherit;
+  align-items: center;
+  list-style: none;
+}
 
-  .logo-info {
-    font-family: poppins, sans-serif;
-    font-size: 18pt;
-    font-weight: 700;
-    line-height: 74%;
-    text-align: left;
-    left: 12px;
-    top: 10px;
-    vertical-align: center;
-    text-decoration: none;
-    color: #6070ff;
-    margin: 0;
-    padding: 0;
-  }
+.header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  color: #344563;
+  padding: 16px;
+}
 
-  .logo-vector-btn {
-    border: 0;
-    background-color: inherit;
-  }
+.header-bar {
+  display: flex;
+  justify-content: space-between;
+  padding-left: 32px;
+  align-items: center;
+  height: 50px;
+}
 
-  .welcome {
-    color: #344563;
-    max-width: 100%;
-    padding: 16px;
-    margin: auto;
-    gap: 12px;
-    background-color: #fff;
-  }
+.logo-info {
+  font-size: 18pt;
+  font-weight: 700;
+  line-height: 74%;
+  text-align: left;
+  left: 12px;
+  top: 10px;
+  vertical-align: center;
+  text-decoration: none;
+  color: #6070ff;
+  margin: 0;
+  padding: 0;
+  animation: rotation 1s ease-in;
+}
 
-  .container-1 {
-    max-width: 120rem;
-    margin: auto;
-    padding: 12px;
-    display: flex;
-    flex-direction: column;
-    justify-content: space-around;
-    background: url("images/header-shapes\ mobile@2x.svg");
-    background-repeat: no-repeat;
-    background-size: 100%;
-    height: 600px;
+@keyframes rotation {
+  from {
+    transform: rotate(0deg);
   }
 
-  .title {
-    padding-top: 114px;
-    font-family: poppins, sans-serif;
-    font-size: 50px;
-    font-weight: 700;
-    line-height: 52px;
-    letter-spacing: 0.01em;
-    text-align: left;
+  to {
+    transform: rotate(360deg);
   }
+}
 
-  .detailed {
-    padding-top: 12px;
-    font-family: poppins, sans-serif;
-    color: #344563;
-    font-size: 17px;
-    font-weight: 400;
-    line-height: 24px;
-    letter-spacing: 0.01em;
-    text-align: left;
-  }
+.nav-menu {
+  position: fixed;
+  left: -100%;
+  top: 0;
+  width: 100%;
+  height: 98vh;
+  background-color: #6070ff;
+  padding: 16px;
+  text-align: right;
+}
```

Comment:
- [ ] Great effort working on the changes requested by the previous reviewer, but according to the previous reviewer and as stated in the project requirement, you should make sure the mobile menu follows the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, text, image, margins,...).

In the Figma design, the `main page` after the `hamburger` is clicked is showing underneath but in a `blur` manner, but in your case the `main page` is not showing. Please kindly fix that

Your design 👇 

![chepkok](https://user-images.githubusercontent.com/78634841/210310200-82c66313-068e-4020-bec4-b8e7b65c38ca.png)


Figma design 👇 

<img src="https://user-images.githubusercontent.com/74501523/210271315-0d8f1788-1b09-4adf-b464-860e3a983468.png" width="350px">

Reply:
@brainconnect93  I think this is a contradiction because I used the same color in the Figma design as requested by the previous reviewer. Initially, I adjusted the color so that there was a blur for the background elements to appear but I had to change and use the exact color in the Figma. Now you'd like for me to change the color and use which one exactly? Should it be the Figma color or a color that ensures the underneath items are slightly visible? Please specify that to avoid a contradiction.

Reply:
@chepkok3 Kindly use the color given in the Figma template and make sure the `main page` is showing but in a blurry manner. You can achieve this by using the CSS property `backdrop-filter: blur(8px)`.

- [ ] If you check what you have in the previous project before the previous reviewer asked you to make changes, the `main page` was showing but it wasn't `blurred`. we need to achieve both the color and blurring of the main page when the mobile menu is displayed